test/indexer/test_hca_indexer.py

```diff
@@ -596,9 +596,10 @@ def test_indexing_with_skipped_matrix_file(self):
                     file_name = file['name']
                     file_names.add(file_name)
         self.assertEqual(4, len(entities_with_matrix_files))  # a project, a specimen, a cell suspension and a bundle
-        self.assertEqual(aggregate_file_names, file_names)
-        matrix_file_names = {file_name for file_name in file_names if '.zarr!' in file_name}
-        self.assertEqual({'377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0.zarr!.zattrs'}, matrix_file_names)
+        for file_name in aggregate_file_names:
```

Comment:
This could be written more succinctly and clearly using set algebra.

Then again, why does this assertion need to be weakened?

Reply:
This was resolved in F2F convo.